packages/theme-default/src/components/Sidebar/SidebarGroup.tsx

```diff
@@ -177,7 +178,11 @@ export function SidebarGroup(props: SidebarItemProps) {
               />
             ) : (
               // eslint-disable-next-line react/no-array-index-key
-              <div key={index} data-context={item.context}>
+              <div
+                className="rspress-sidebar-group-item"
```

Comment:
Can we make the classNames shorter? such as `rspress-sidebar-item` `rspress-sidebar-menu`